app_dart/lib/src/request_handling/subscription_handler.dart

```diff
@@ -31,16 +31,16 @@ abstract class SubscriptionHandler extends RequestHandler<Body> {
     required this.cache,
     required super.config,
     this.authProvider,
-    required this.topicName,
+    required this.subscriptionName,
   });
 
   final CacheService cache;
 
   /// Service responsible for authenticating this [HttpRequest].
   final AuthenticationProvider? authProvider;
 
-  /// Unique identifier of the PubSub in this cloud project.
-  final String topicName;
+  /// Unique identifier of the PubSub subscription in this cloud project.
+  final String subscriptionName;
```

Comment:
This is the first time Cocoon is having a second subscription to a topic. I swapped it to the subscription name, but it really doesn't matter as long as it's unique since it's for a write-lock

Reply:
Not in this PR, could we have any internal doc about subscription/topic, like creation from GCP side, how to connect those topics from cocoon side, etc? Or we already have them?